app-frontend/src/app/components/scenes/sceneImportModal/sceneImportModal.html

```diff
@@ -81,6 +79,27 @@ <h3 class="no-margin">Import your Scenes</h3>
           <div class="text-center"><strong>DropBox</strong></div>
         </rf-box-select-item>
       </div>
+      <div ng-if="$ctrl.importType === 'S3'">
+        <p>
+          Enter the the S3 bucket and prefix to indicate where your imagery is located.
+          The import process will import any imagery found in that location into Raster Foundry.
+        </p>
+        <form>
+          <div class="form-group">
+            <label for="name">S3 Bucket</label>
+            <input id="name" type="url" class="form-control"
+                  placeholder="S3 Bucket" ng-model="$ctrl.s3Config.bucket">
```

Comment:
I think the placeholder text should be closer to what users actually need to put in (e.g. `s3://<bucket>/<prefix>/`) otherwise it isn't clear.

Alternative, two fields (one for bucket and one for prefix -- the prefix being optional).